### PR TITLE
BHBC-1342

### PR DESCRIPTION
**Pull request by cgarrettjones**
# Overview

## Links to jira tickets

- [BHBC-1342](https://quartech.atlassian.net/browse/BHBC-1342)

## This PR contains the following changes

- addition of survey_summary_general table
- addition of survey_summary_detail table

## This PR contains the following types of changes

- [ ] New feature (change which adds functionality)
- [ ] Enhancement (improvements to existing functionality)
- [ ] Bug fix (change which fixes an issue)
- [ ] Misc cleanup / Refactoring / Documentation
- [ ] Version change

## Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### General

- [ ] I have performed a self-review of my own code

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Screenshots

Please add any relevant UI screenshots, if applicable.

### CODE REVIEW

**Review by sdevalapurkar — COMMENTED** [review]
code changes lgtm, haven't tested locally

File: database/src/migrations/release.0.26/biohub.sql [hunk 1]
```diff
@@ -1,3575 +1,3746 @@
...
+-- TABLE: survey_summary_detail 
+--
+
+CREATE TABLE survey_summary_detail(
+    survey_summary_detail_id       integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    id                             integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    survey_summary_general_id      integer           NOT NULL,
+    parameter                      varchar(100)      NOT NULL,
+    parameter_value                character(10)     NOT NULL,
+    parameter_methodv              varchar(25),
```
Comment (NickPhura) [inline database/src/migrations/release.0.26/biohub.sql]: Should this name (`parameter_methodv`) have that trailing `v`?

File: database/src/migrations/release.0.26/biohub.sql [hunk 2]
```diff
@@ -1,3575 +1,3746 @@
...
+COMMENT ON TABLE survey_spatial_component IS 'Survey spatial component persists the various spatial components that a survey may include. Examples are block and transect spatial artifacts.'
+;
+
+-- 
+-- TABLE: survey_summary_detail 
+--
+
+CREATE TABLE survey_summary_detail(
+    survey_summary_detail_id       integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    id                             integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
```
Comment (NickPhura) [inline database/src/migrations/release.0.26/biohub.sql]: Is this `id` intentional? Or a duplicate of `survey_summary_detail_id`?

**Review by anissa-agahchen — COMMENTED** [review]
Does parameter in the survey_summary_detail map to 'density', 'total abundance', 'bull:cow ratio', etc?

File: database/src/migrations/release.0.26/biohub.sql (hunk 2 above)
Comment (cgarrettjones) [reply database/src/migrations/release.0.26/biohub.sql]: some wierd-ness going on... will fix.

**Review by anissa-agahchen — APPROVED** [review]
Tested.  Works fine.

**Pull request merged**